tests/unit/amo/test_tracking.js

```diff
@@ -233,6 +233,22 @@ describe(__filename, () => {
         }).toThrowError(/action is required/);
       });
 
+      // This is a way to check how many times window.ga was called with the
+      // arguments that signify that it was sending an event.
+      const countSendEventCalls = (spy) => {
+        return spy
+          .getCalls()
+          .filter(
+            (callObj) =>
+              callObj.firstArg === 'send' &&
+              Object.prototype.hasOwnProperty.call(
```

Comment:
I had to use this `call` syntax because ESLint didn't like the more simple `callObj.lastArg.hasOwnProperty('hitType')` syntax.

Reply:
what did ESLint say by chance? maybe we can disable it if it's non-sense

Reply:
It's this error: https://eslint.org/docs/rules/no-prototype-builtins. I'm not sure if we want to disable it for the whole project. The comment under "When Not To Use It" makes it sound like we're probably okay disabling it, but what do you think?

Reply:
We could disable it for this file or only this part of the code maybe?

Reply:
I'm not sure that the compliant code is really that terrible, I just wanted to explain why I did it in the PR comment. I'm not sure that adding a comment in order to be able to use the simpler code is actually any better than just using the suggested code, which is why I chose to write it that way. WDYT?

Reply:
I don't mind but if ESLint forces us to use something that we don't for no real value-added, we could simply mute it. Choose what you like the most!